README.md

```diff
@@ -22,4 +22,4 @@ A .Net dynamic fake framework for creating all types of fake objects, mocks, stu
 * [Package](https://nuget.org/packages/FakeItEasy "FakeItEasy on NuGet")
 
 -
-FakeItEasy logo designed by Vanja Pakaski.
+FakeItEasy logo designed by [@vanpak](https://github.com/vanpak).

```

Comment:
Not "by [Vanja Pakaski](https://github.com/vanpak)"?

I'm not fussy, but thought, "why not use her name?". Makes her sound more like a human.